### PR TITLE
feat: Assets portfolio

### LINKED ISSUE #115: Redesign: Portfolio screen
https://www.figma.com/file/bZLZ258L7ZQqGLtqsHrmsE/Starport-Mobile-App-Template?node-id=0%3A1

image

### PR DESCRIPTION
**Pull request by wal33d006**
Closes #115 

Figma:
Assets portfolio

Actual:
Simulator Screen Shot - iPhone 12 Pro - 2021-11-18 at 22 08 49

### CODE REVIEW

File: starport_template/lib/pages/assets_portfolio_page.dart [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  bool get isError => StarportApp.walletsStore.isBalancesLoadingError;
+
+  @override
+  void initState() {
+    super.initState();
+    _fetchWalletBalances();
+  }
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (andrzejchm) [inline starport_template/lib/pages/assets_portfolio_page.dart:39]: the widget structure is a bit deep and complex, could we extract some to separate Widgets/ builder methods for readability?

File: starport_template/lib/pages/wallet_details_page.dart [hunk 2]
```diff
@@ -88,27 +87,27 @@ class _WalletDetailsPageState extends State<WalletDetailsPage> {
...
-    );
-    if (result == true) {
-      StarportApp.walletsStore.getBalances(widget.walletInfo.address);
-    }
-  }
+  // Future<void> _openSendMoneySheet(Denom denom) async {
+  //   final result = await showModalBottomSheet(
+  //     context: context,
+  //     builder: (context) => SafeArea(
+  //       child: SendMoneySheet(
```
Comment (andrzejchm) [inline starport_template/lib/pages/wallet_details_page.dart:103]: let's add a `TODO` for deleting `SendMoneySheet` aswell

File: starport_template/lib/pages/assets_portfolio_page.dart (hunk 1 above)
Comment (wal33d006) [reply starport_template/lib/pages/assets_portfolio_page.dart:39]: Updated

File: starport_template/lib/pages/wallet_details_page.dart (hunk 2 above)
Comment (wal33d006) [reply starport_template/lib/pages/wallet_details_page.dart]: Updated

**Review by andrzejchm — APPROVED** [review]
🚀

**Pull request merged**